### ISSUE TITLE
Comparison to Saturn

### ISSUE THREAD

**Issue by dariotommasini** [snap-stanford/UCE#62]
Hi @Yanay1 , 

Thank you for the great tool! I am curious about when one might use UCE and when one might use Saturn, specifically for cross-species integration. Have you seen whether one outperforms the other and what conditions might those be? For instance, is one better for more closely related species and another better for distantly related species? Would love to get your insight on this. 

Thanks!

**Comment by Yanay1** [comment]
We checked performance for label transfer comparing UCE to SATURN on the 4 datasets for species that UCE wasn't trained on in the paper (green monkey lung, chicken retina, chicken heart and naked mole rat spleen), and actually UCE outperformed SATURN for label transfer from a human tissue paired dataset on 3 / 4 of them. UCE also outperformed SAMap on 3/4 of the datasets for the same task. 

This was a nice surprise for use since 1. UCE was working on species that weren't even in the training dataset and it worked with no fine tuning at all 2. UCE doesn't take in cell types like SATURN and still outperformed it.

I think because it is zero shot and doesn't need cell types, or even any reference dataset, UCE is good to run as a first check. If you are confident in your cell types and primarily interested in seeing how they transfer to a new species, SATURN is a good choice to run also.
